.github/workflows/chainloop.yml

```diff
@@ -26,53 +29,45 @@ jobs:
           sparse-checkout: |
             .chainloop.yml
 
-      - uses: sigstore/cosign-installer@v3.1.1
-      
-      - name: Install Chainloop CLI
+      - name: Install Chainloop CLI, Labs and Cosign
         run: |
-          if [ -n "${CHAINLOOP_VERSION}" ]; then
-             curl -sfL https://docs.chainloop.dev/install.sh | bash -s -- --version v${CHAINLOOP_VERSION}
-          else
-            curl -sfL https://docs.chainloop.dev/install.sh | bash -s
-          fi
-          
+          bash -c "$(curl -fsSL https://raw.githubusercontent.com/chainloop-dev/labs/dev/tools/src/lib/chainloop.sh) ; install_chainloop_labs dev"
+          . ~/chainloop.sh
+          install_chainloop_cli_and_cosign
+
       - name: Download all workflow run artifacts
         uses: actions/download-artifact@v3
-      
+
       - name: Initialize Attestation
-        run: chainloop attestation init # --contract-revision 2
+        run: . ~/chainloop.sh && chainloop_attestation_init && ls -las
+
+      - name: logs
+        run: . ~/chainloop.sh && chainloop_collect_logs_for_github_jobs
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
I think there is already a GITHUB_ACTION environment variable available. So you don't need to create a different one.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow

Reply:
Thanks for catching it!